### PR TITLE
Update intro.md

### PR DESCRIPTION
**Pull request by yang-ruoxi**
suggesting to add a section for JOSS partnership

### CODE REVIEW

**Comment by lwasser** [comment]
@all-contributors please add @yang-ruoxi for code, review

File: about/intro.md [hunk 1]
```diff
@@ -129,3 +129,9 @@ pyOpenSci Peer Review Badge
 This badge is added by the package author after the package
 has successfully completed review and ideally links to the specific GitHub issue
 where the tool was reviewed. [See this example from devicely, one of our accepted pyOpenSci ecosystem packages](https://github.com/hpi-dhc/devicely).
+
+
+### Partnership with JOSS
```
Comment (willingc) [inline about/intro.md:134]:
```suggestion
### Partnership with JOSS
```

File: about/intro.md [hunk 2]
```diff
@@ -129,3 +129,9 @@ pyOpenSci Peer Review Badge
 This badge is added by the package author after the package
 has successfully completed review and ideally links to the specific GitHub issue
 where the tool was reviewed. [See this example from devicely, one of our accepted pyOpenSci ecosystem packages](https://github.com/hpi-dhc/devicely).
+
+
+### Partnership with JOSS 
+
+pyOpenSci collaborates with organizations that support the scientific community, for example, Journal of Open Source Software (JOSS).
```
Comment (willingc) [inline about/intro.md:136]:
```suggestion
pyOpenSci collaborates with organizations that support the scientific community, for example, Journal of Open Source Software (JOSS).
```

**Review by willingc — APPROVED** [review]
This looks good with a couple of fixes for trailing whitespace.

@lwasser Let me know if this PR still makes sense to merge.

**Comment by lwasser** [comment]
i missed merging this - thank you for the ping @willingc we can merge now!

**Pull request merged**